src/platform/datapath_raw_xdp.c

```diff
@@ -923,20 +930,37 @@ CxPlatDpRawInterfaceRemoveRules(
     CxPlatLockRelease(&Interface->RuleLock);
 }
 
+
+_IRQL_requires_max_(PASSIVE_LEVEL)
+size_t
+CxPlatDpRawGetDapathSize(
+    _In_opt_ const CXPLAT_DATAPATH_CONFIG* Config
+    )
+{
+    const uint32_t WorkerCount =
+        (Config && Config->RawDataPathProcList) ? Config->RawDataPathProcListLength : 1;
+    return sizeof(XDP_DATAPATH) + (WorkerCount * sizeof(XDP_WORKER));
+}
+
 _IRQL_requires_max_(PASSIVE_LEVEL)
 QUIC_STATUS
 CxPlatDpRawInitialize(
     _Inout_ CXPLAT_DATAPATH* Datapath,
     _In_ uint32_t ClientRecvContextLength,
-    _In_opt_ CXPLAT_DATAPATH_CONFIG* Config
+    _In_opt_ const CXPLAT_DATAPATH_CONFIG* Config
     )
 {
     XDP_DATAPATH* Xdp = (XDP_DATAPATH*)Datapath;
     QUIC_STATUS Status;
 
-    CxPlatXdpReadConfig(Xdp, Config);
-    CxPlatDpRawGenerateCpuTable(Datapath);
+    uint16_t DefaultCore = (uint16_t)(CxPlatProcMaxCount() - 1);
+    const uint16_t* PollingCores =
```

Comment:
>PollingCores [](http://example.com/codeflow?start=20&length=12)

It seems that the term processor/proc is more widely used in QUIC and XDP than cores.